Python/optimizer_cases.c.h

```diff
@@ -406,10 +417,11 @@
                 // replace opcode with constant propagated one and update tests!
             }
             else {
-                res = sym_new_type(ctx, &PyFloat_Type);
+                res = sym_new_type(ctx, &PyFloat_Type);stack_pointer += -2;
```

Comment:
Here too. I'm going to stop pointing these out since fixing the code generator will probably fix them all.